index.bs

```diff
@@ -77,16 +88,16 @@ beyond those described in the Generic Sensor API [[!GENERIC-SENSOR]].
 Model {#model}
 =====
 
-The magnetometer's associated {{Sensor}} subclass is the {{Magnetometer}} class.
-
-The Magnetometer has a <a>default sensor</a>, which is the device's main magnetomer sensor.
+The magnetometer has two associated {{Sensor}} subclasses, {{Magnetometer}} and {{RawMagnetometer}}.
```

Comment:
https://github.com/w3c/magnetometer/issues/16#issuecomment-258535158 ;
We decided to split into 2 separate interfaces. 

Calibrated = raw + soft iron + hard iron;
Uncalibrated = raw +  soft iron 

As you know, its not a simple _plus_ but some transform matrix applied..